game/static/game/js/level_editor.js

```diff
@@ -2094,25 +2091,20 @@ ocargo.LevelEditor = function() {
                 }
             }
 
-            if(!connected) {
-                return false;
-            }
+            return !connected && notAlreadyOccupied(sourceNode, controlledNode);
```

Comment:
Bah. I'll kill the PR when I'm next near an internet connection. Brain is
really failing me of late :-(

On Friday, 6 May 2016, Oliver Jeeves <notifications@github.com> wrote:

> In game/static/game/js/level_editor.js
> <https://github.com/ocadotechnology/rapid-router/pull/771#discussion_r62299712>
> :
>
> > @@ -2094,25 +2091,20 @@ ocargo.LevelEditor = function() {
> >                  }
> >              }
> >
> > -            if(!connected) {
> > -                return false;
> > -            }
> > +            return !connected && notAlreadyOccupied(sourceNode, controlledNode);
>
> Previously is connected was false, we returned false. Now, if it's true,
> we'll return false.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ocadotechnology/rapid-router/pull/771/files/050963df2e95ace4425f8b02b650b447c812f8d0#r62299712>
>